config/preset/yazi.toml

```diff
@@ -2,6 +2,7 @@
 sort_by      = "modified"
 sort_reverse = true
 show_hidden  = false
+sort_dir_first = true
```

Comment:
It's better to move this line up one, so that the `sort_*` is in the same group.

Reply:
Of course!